tests/lvmo/test_lvm_overprovisioning.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import re
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1, skipif_lvm_not_installed
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, acceptance
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import LVM
+from ocs_ci.ocs.exceptions import ResourceWrongStatusException
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@acceptance
+@skipif_lvm_not_installed
+@skipif_ocs_version("<4.11")
+class TestLvmOverProvisioning(ManageTest):
+    """
+    Test lvm snapshot bigger than disk
+
+    """
+
+    @pytest.mark.parametrize(
+        argnames=["volume_mode", "volume_binding_mode"],
+        argvalues=[
+            pytest.param(
+                *[
+                    constants.VOLUME_MODE_FILESYSTEM,
+                    constants.IMMEDIATE_VOLUMEBINDINGMODE,
+                ],
+                marks=pytest.mark.polarion_id("OCS-4469"),
+            ),
+            pytest.param(
+                *[constants.VOLUME_MODE_BLOCK, constants.IMMEDIATE_VOLUMEBINDINGMODE],
+                marks=pytest.mark.polarion_id("OCS-4469"),
+            ),
+        ],
+    )
+    def test_lvm_over_over_provisioning(
+        self,
+        volume_mode,
+        volume_binding_mode,
+        project_factory,
+        lvm_storageclass_factory,
+        pvc_factory,
+    ):
+        """
+        test create delete snapshot
+        .* Check overprovisioning value
+        .* Check thin pool size
+        .* Create PVCs till you reach overprovisioning
+        .* Check for error
+
+        """
+
+        access_mode = constants.ACCESS_MODE_RWO
+
+        lvm = LVM(fstrim=True, fail_on_thin_pool_not_empty=True)
+        thin_pool_size = lvm.get_thin_pool1_size()
+
+        if "t" in thin_pool_size:
```

Comment:
what if the think pool size is in GB or MB?

Reply:
In MB? Not a real life scenario. The else is taking the GB option. What can be done is add to all OS commands (lvs, pvs, vgs) `--unit g` and then all of the outputs will have G for sure.

Reply:
adding `--unit g` makes sense here

Reply:
Added --unit g to all os calls. Removed parsing the unit from the test.